src/gl/contourFeature.js

```diff
@@ -0,0 +1,305 @@
+//////////////////////////////////////////////////////////////////////////////
+/**
+ * Create a new instance of contourFeature
+ *
+ * @class
+ * @extends geo.contourFeature
+ * @returns {geo.gl.contourFeature}
+ */
+//////////////////////////////////////////////////////////////////////////////
+geo.gl.contourFeature = function (arg) {
+  'use strict';
+
+  if (!(this instanceof geo.gl.contourFeature)) {
+    return new geo.gl.contourFeature(arg);
+  }
+  arg = arg || {};
+  geo.contourFeature.call(this, arg);
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * @private
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  var m_this = this,
+      s_exit = this._exit,
+      m_textureUnit = 7,
```

Comment:
No particular reason, except that unit 0 is used in other features.

We really should have a function that tracks usage of them and issues the
next available one.

On Tue, Jun 2, 2015 at 2:29 PM, Aashish Chaudhary <notifications@github.com>
wrote:

> In src/gl/contourFeature.js
> <https://github.com/OpenGeoscience/geojs/pull/395#discussion_r31554810>:
>
> > +  'use strict';
> > +
> > +  if (!(this instanceof geo.gl.contourFeature)) {
> > +    return new geo.gl.contourFeature(arg);
> > +  }
> > +  arg = arg || {};
> > +  geo.contourFeature.call(this, arg);
> > +
> > +  ////////////////////////////////////////////////////////////////////////////
> > +  /**
> > +   * @private
> > +   */
> > +  ////////////////////////////////////////////////////////////////////////////
> > +  var m_this = this,
> > +      s_exit = this._exit,
> > +      m_textureUnit = 7,
>
> why 7? I am assuming just to be on the safe side?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenGeoscience/geojs/pull/395/files#r31554810>.
>



-- 
David Manthey
R&D Engineer
Kitware Inc.
(518) 881-4439